wrtbwmon

```diff
@@ -76,6 +84,9 @@ lookup()
     if [ -n "$DO_RDNS" -a -z "$USER" -a "$IP" != "NA" -a -n "$nslookup" ]; then
 	USER=`$nslookup $IP $DNS | awk '!/server can/{if($4){print $4; exit}}' | sed -re 's/[.]$//'`
     fi
+    if [ -n "$DO_RDNS" -a -z "$USER" ]; then
```

Comment:
Lookup is only called from the `publish` CLI command. By that point, we should already have parsed /proc/net/arp. If this feature isn't working, let's fix it separately.